coalib/bearlib/abstractions/Linter.py

```diff
@@ -363,8 +358,7 @@ def process_output_regex(
             Processes the executable's output using a regex.
```

Comment:
you should mention in both places that this may be called twice with stderr *and* stdout if specified so